event-log/README.md

```diff
@@ -0,0 +1,243 @@
+# event-log
+
+This is a microservice which provides CRUD operations for Event Log DB.
+
+## API
+
+| Method | Path                                     | Description                                                    |
+|--------|------------------------------------------|----------------------------------------------------------------|
+|  POST  | ```/events```                            | Creates an event with a `NEW` status                           |
+|  PATCH | ```/events```                            | Changes events' data by applying the given patch               |
+|  GET   | ```/events/latest```                     | Finds events for all the projects with the latest `event_date` |
+|  PATCH | ```/events/:id/projects/:id/status```    | Updates event status                                           |
+|  GET   | ```/events/projects/:id/status```        | Finds processing status of events belonging to a project       |
+|  POST  | ```/events/subscriptions?status=READY``` | Adds a subscription for the events                             |
```

Comment:
I would suggest to make the subscription a top-level entity, so just `POST /subscriptions`. Details of the subscriptions can be in the body.

Reply:
Great idea!